liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -322,3 +322,14 @@ error.install.dependency.app.missing=CWWKM2180E: The application dependency for
 error.install.dependency.app.missing.explanation=The application dependency cannot be found in Maven local repository.
 error.install.dependency.app.missing.useraction=Rebuild the project by running mvn install again.
 
+info.install.license=CWWKM2181I: Installing Liberty production license {0}.
+info.install.license.explanation=Installing Liberty production license to upgrade the Liberty runtime installation to the new edition and license terms.
+info.install.license.useraction=No action is required.
+
+info.install.license=CWWKM2182W: Liberty production license jar file {0} may be corrupted.
```

Comment:
This is using the same property key as the previous message.  Should this be warn.install.license?